image-tools-interface.md

```diff
@@ -0,0 +1,49 @@
+# Image Tools Interface
+
+The below document describes an interface for the OCI Image Tools project as desired by the OCI Certification Program working group (Cert WG.)
+
+Goals:
+* An engineer can verify a container image complies with the OCI Image Spec (https://github.com/opencontainers/image-spec/releases) 
+
+Inputs:
+* (REQUIRED): Path or reference to a container image
+* (OPTIONAL): If the user would like verbose output
+
+Outputs:
+* Number of cases run
+* Number of cases passed
+* Number of cases failed
+* List of cases failed
+* Version of OCI Image Tools
```

Comment:
and maybe git commit of the tool? 
and version of the OCI spec tested against.
And the timestamp the validation just ran.
Also, presumably this would need to output the digests of the objects tested.

With this data points in a structured output (i.e. xml, json, etc) then it could be more deterministically used for badges, etc.